src/main/java/com/tenduke/events/api/model/data/OrganizationFields.java

```diff
@@ -0,0 +1,23 @@
+package com.tenduke.events.api.model.data;
+
+import com.fasterxml.jackson.annotation.JsonInclude;
+
+/**
+ * Event data fields for organization events.
+ * @author roope
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+public interface OrganizationFields {
```

Comment:
Should probably be `OrganizationInvitationFields` since this has field named `invitationId`?